ra/ra.go

```diff
@@ -2319,6 +2319,10 @@ func (ra *RegistrationAuthorityImpl) GenerateOCSP(ctx context.Context, req *rapb
 		return nil, err
 	}
 
+	if status.Status == string(core.OCSPStatusNotReady) {
+		return nil, berrors.InternalServerError("issuance failed after linting")
```

Comment:
I think it's probably worth documenting why we return a 500 here. It's slightly surprising, given that no error occurred _during this request_. But an error did occur during some previous work, we expect these responses to be vanishingly rare, and we really want to know when they _do_ happen, so 500 is appropriate.